.github/workflows/test-workflow.yml

```diff
@@ -2,7 +2,7 @@ name: Hello World 워크플로우
 
 on:
   push:
-    branches: [main]
+    branches: [test-branch] # main 대신 test-branch에서만 실행
```

Comment:
The PR changes the `branches` trigger from `main` to `test-branch`. Ensure that this change doesn't unintentionally prevent the workflow from running on the `main` branch if that was previously intended. A comment explaining the reasoning behind this change within the workflow file itself could be beneficial for future maintainers.